src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -1,5 +1,6 @@
 <div
   class="c-mini-shelf"
+  [ngClass]="{'transparent-red-zebra-opacity': isMediaUnavailable}"
```

Comment:
Being a common class (in this case for MiniShelf and OffsetablePiece components) a descriptive (what it does) naming might suit better, so it can be used in different contexts as needed (also relates to https://github.com/tv2/ng-sofie/pull/104#discussion_r1491268233).